package.json

```diff
@@ -151,7 +151,7 @@
     "lint": "eslint ./",
     "lint-fix": "eslint ./ --fix",
     "tsc": "tsc --pretty --noEmit",
-    "prepare": "husky install"
+    "prepare": "husky install || true"
```

Comment:
What is this for I understand its running husky before the packages are packed but, this isn't an npm package?

Reply:
deployment was failing on this line. Adding the "|| true" is actually recommended in the husky How to https://typicode.github.io/husky/how-to.html